demo/helm/rook-operator/README.md

```diff
@@ -0,0 +1,110 @@
+rook-operator
```

Comment:
have you been able to test this on 1.5 and 1.6 already? The test matrix could grow pretty quickly here